readme.md

```diff
@@ -195,6 +195,11 @@ Run a quick demo would be like:
 ```
 python3 demo.py --config-file configs/wearmask/darknet53.yaml --input ./datasets/wearmask/images/val2017 --opts MODEL.WEIGHTS output/model_0009999.pth
 ```
+Run a quick demo with weights and biases logging
```

Comment:
Maybe make this more compelling, something like

"Run a quick demo to upload and explore your YOLOv7 prediction with [Weights & Biases](https://wandb.ai/site). [See here for an example](INSERT LINK TO PUBLIC RUN WITH YOLOv7 INFERENCE TABLE)

Also note that we use "&" instead of "and" as well as capitalisation in "Weights & Biases"